INSTALL.md

```diff
@@ -99,13 +114,43 @@ The initialization command will prompt for the following information.
    prompts for its password.  This information will be needed to later connect
    to Accumulo.
 
-Multiple Nodes
---------------
+## Run Accumulo
+
+There are two primary methods for running Accumulo:
+
+1. Run individual Accumulo services using `accumulo-service`. Useful if you are
+   using a cluster management tool (i.e Ansible, Salt, etc) or init.d scripts to
+   start Accumulo.
 
-Skip this section if running Accumulo on a single node.  Accumulo has
-coordinating, monitoring, and worker processes that run on specified nodes in
-the cluster.  The following files should be populated with a newline separated
-list of node names.  Must change from localhost.
+2. Run an Accumulo cluster on one or more nodes using `accumulo-cluster`. Useful for
```

Comment:
It would be useful to mention that accumulo-cluster utilizes accumulo-service, and inherits the added functionality/side-effects of that script.